core/src/main/java/tc/oc/pgm/flag/state/Respawning.java

```diff
@@ -54,7 +56,9 @@ protected Duration getDuration() {
   public void enterState() {
     super.enterState();
 
-    if (Duration.ZERO.equals(respawnTime)) return;
+    if (Duration.ZERO.equals(respawnTime) || this.flag.getMatch().getPhase() == MatchPhase.FINISHED)
```

Comment:
```suggestion
    if (Duration.ZERO.equals(respawnTime) || this.flag.getMatch().isFinished())
```
Also, should it be `!match.isRunning()`  instead? i guess it can't happen before the match so its prob not relevant